### PR TITLE
Added where can i find help? to faq

### PR DESCRIPTION
**Pull request by ankushbhardwaj408**
Hi @kathatherine i have added the "Where can i find help?" to the faq section. Issue #53 
Screenshot 2023-04-03 220154

### CODE REVIEW

**Comment by travishathaway** [comment]
Hi @ankushbhardwaj408,

Thanks for taking the initiative to pick up this issue.

I have a suggestion to improve the answer you submitted. On the community page, we have a "[How to reach us](https://conda-dot-org.netlify.app/community#how-to-reach-us)" section. Could you please link to that in your answer? Not only does it list the element chat rooms but also links to the conda Discourse server and our GitHub organizations.

**Comment by ankushbhardwaj408** [comment]
HI @travishathaway 
Thanks for the suggestions. I am working on it.

**Comment by ankushbhardwaj408** [comment]
HI @travishathaway i have made the suggested changes. Please review it. #84 
Screenshot 2023-04-04 205258

File: learn/faq.md [hunk 1]
```diff
@@ -71,3 +71,12 @@ See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/mai
 ## I see a problem with the website. How do I report it?
 
 Please create an [issue](https://github.com/conda-incubator/conda-dot-org/issues) in GitHub to report any problems with the website. We also welcome pull requests!
+
+## Where can I find help?
+
+If you want some help you can reach out to us and get the necessary help.
```
Comment (travishathaway) [inline learn/faq.md:77]:
```suggestion
If you need help with any aspect of the conda ecosystem, feel free to reach out to us via any of our online channels.
```

File: learn/faq.md [hunk 2]
```diff
@@ -71,3 +71,12 @@ See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/mai
...
+Here are the social profiles of conda : [Element](https://app.element.io/#/room/#conda_conda:gitter.im),  [Discourse](https://conda.discourse.group/), [Github](https://github.com/conda-incubator).
```
Comment (travishathaway) [inline learn/faq.md:78]:
```suggestion
```

File: learn/faq.md [hunk 3]
```diff
@@ -71,3 +71,12 @@ See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/mai
...
+
+For more information [visit](http://localhost:3000/community#how-to-reach-us) here.
```
Comment (travishathaway) [inline learn/faq.md:80]:
```suggestion
For more information read [How to reach us](/community#how-to-reach-us) on the [Community](/community) page.
```

**Review by travishathaway — CHANGES_REQUESTED** [review]
Hi @ankushbhardwaj408,

Please see my suggestions for improvement.

Please never use "https://localhost" links on any page for this website. This will not work. Instead, just use the absolute path (e.g. "/community").

**Comment by ankushbhardwaj408** [comment]
Thanks @travishathaway for the suggestions.

File: learn/faq.md [hunk 4]
```diff
@@ -71,3 +71,11 @@ See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/mai
 ## I see a problem with the website. How do I report it?
 
 Please create an [issue](https://github.com/conda-incubator/conda-dot-org/issues) in GitHub to report any problems with the website. We also welcome pull requests!
+
+## Where can I find help?
+
+If you need help with any aspect of the conda ecosystem, feel free to reach out to us via any of our online channels.
+
+For more information read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.
```
Comment (travishathaway) [inline learn/faq.md:79]:
```suggestion
For more information, read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.
```

File: learn/faq.md [hunk 5]
```diff
@@ -71,3 +71,11 @@ See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/mai
 ## I see a problem with the website. How do I report it?
 
 Please create an [issue](https://github.com/conda-incubator/conda-dot-org/issues) in GitHub to report any problems with the website. We also welcome pull requests!
+
+## Where can I find help?
+
+If you need help with any aspect of the conda ecosystem, feel free to reach out to us via any of our online channels.
+
+For more information, read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.
```
Comment (travishathaway) [inline learn/faq.md:79]:
```suggestion
For more information, read [How to reach us](/community/#how-to-reach-us) on the [Community](/community) page.
```

**Review by travishathaway — APPROVED** [review]

**Pull request merged**